submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionDocDataRepositoryTest.kt

```diff
@@ -47,17 +49,17 @@ internal class SubmissionDocDataRepositoryTest(
 ) {
 
     @BeforeEach
-    fun beforeEach() {
-        testInstance.deleteAll()
+    fun beforeEach() = runBlocking {
```

Comment:
I understand `runTest` is equivalent to `runBlocking` but to be used in a test context since it handles the exceptions differently

```
If the created coroutine completes with an exception, then this exception will be thrown at the end of the test
```

So, I guess it's ok to use it here as well, just to avoid mixing those two. WDYT?

Reply:
to be honest I am not sure but I think given this is not a test we should not use annotation.  runTest Documentaiton said `If the created coroutine completes with an exception, then this exception will be thrown at the end of the test.` I think in before each if something fail we want inmediante fail and also error should not propagetd to any test.